WebRoot/js/zimbraMail/briefcase/view/ZmDetailListView.js

```diff
@@ -48,6 +48,7 @@ ZmDetailListView = 	function(parent, controller, dropTgt) {
 				  controller:controller, headerList:headerList, dropTgt:dropTgt};
 	ZmBriefcaseBaseView.call(this, params);
 
+	this._listDiv.setAttribute("aria-label", ZmMsg.documentList);
```

Comment:
same as above if DwtListView can set this based on some passed param then we don't have to repeat this statement in all list views

Reply:
Done.